djangocms_text_ckeditor/models.py

```diff
@@ -25,6 +25,9 @@ class AbstractText(CMSPlugin):
     body = models.TextField(_("body"))
 
     search_fields = ('body',)
+
+    # This property is deprecated. And will be removed in a future release.
+    # It should be set on the Plugin, not the model.
     disable_child_plugins = True
```

Comment:
I don't think we need a deprecation period for this. This is virtually not working now, right?
Anyway the API is on the CMS side, text plugin is just a consumer, thus we can drop this without further warnings